### ISSUE TITLE
Remote shells (via network)

### ISSUE THREAD

**Issue by almarklein** [pyzo/pyzo#4]
Originally reported by: **Almar Klein (Bitbucket: [almarklein](http://bitbucket.org/almarklein), GitHub: [almarklein](http://github.com/almarklein))**

---

_Imported from [iep.googlecode.com issue #4](https://code.google.com/p/iep/issues/detail?id=4), reported by almar.klein@gmail.com_

Since IEP communicates with its shells via a socket, it is possible to run and control shells on another computer. However, this would need quite a bit of infrastructure (the remote host needs a kind of server that creates iep-shell-processes on requests).

Such functionality would also open doors to parallel computing, maybe dividing tasks over multiple shells ... etc.

---
- Bitbucket: https://bitbucket.org/iep-project/iep/issue/4

**Comment by vaibhawc** [comment]
Any updates on this one?
I had been exploring to run pyzo as an interface that connects and shows remote scripts on editor and uses remote python for shell.
I had to ultimately start using jupyter for similar stuff but I am still fond this not-so-popular software.
Let me know if I can be of help in this regard.

**Comment by almarklein** [comment]
No. Still on my whishlist, but will likely not be done before a major refactor / rewrite that I want to do sometime ... when I can make the time for it ...
